srcpkgs/freetube/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'freetube'
+
+pkgname=freetube
+_pkgname=FreeTube
+version=0.19.2
+revision=1
+archs="x86_64"
+short_desc='Open source desktop YouTube player built with privacy in mind'
+license='AGPL-3.0-or-later'
+hostmakedepends="gcc"
+depends='electron24'
+makedepends='yarn'
+maintainer="budrz55 <budroz135@tutanota.com>"
+homepage=https://freetubeapp.io
+distfiles=(https://github.com/FreeTubeApp/FreeTube/archive/v$version-beta.tar.gz
```

Comment:
`xbps-src` at the moment doesn't support arrays, so it only takes the first argument. It doesn't matter, since with `xbps-src` you don't need to list assets in `$FILESDIR` in `distfiles`.